--- origsrc/lxde-common-0.4.1/icon-theme/Makefile.am	2009-04-14 12:37:45.000000000 -0500
+++ src/lxde-common-0.4.1/icon-theme/Makefile.am	2009-06-30 00:23:35.645557200 -0500
@@ -9,7 +9,7 @@ uninstall-hook:
 
 install-data-hook:
 	$(mkdir_p) $(DESTDIR)$(datadir)/icons
-	cp -raf --no-preserve=ownership nuoveXT2 $(DESTDIR)$(datadir)/icons
+	cp -raf --no-preserve=ownership $(srcdir)/nuoveXT2 $(DESTDIR)$(datadir)/icons
 	$(LN_S) preferences-desktop.png $(DESTDIR)$(datadir)/icons/nuoveXT2/128x128/categories/xfce4-settings.png
 	$(LN_S) preferences-desktop.png $(DESTDIR)$(datadir)/icons/nuoveXT2/128x128/categories/gnome-settings.png
 	$(LN_S) preferences-desktop.png $(DESTDIR)$(datadir)/icons/nuoveXT2/128x128/categories/kcontrol.png
