--- origsrc/lxde-common-0.5.5/Makefile.am	2011-07-29 09:04:03.000000000 -0500
+++ src/lxde-common-0.5.5/Makefile.am	2012-08-17 16:14:51.360077900 -0500
@@ -9,6 +9,9 @@ xsession_DATA = LXDE.desktop
 defaultsdir=$(sysconfdir)/xdg/lxsession/LXDE
 defaults_DATA = autostart desktop.conf
 
+applicationsdir=$(datadir)/applications
+applications_DATA = lxde-logout.desktop
+
 wallpapersdir=$(datadir)/lxde/wallpapers
 wallpapers_DATA= \
 	wallpapers/lxde_blue.jpg \
